src/energy.cpp

```diff
@@ -9,7 +9,7 @@ namespace Energy {
 EwaldData::EwaldData(const json &j) {
     alpha = j.at("alpha");                          // damping-parameter
     r_cutoff = j.at("cutoff");                      // real space cut-off
-    k_cutoff = j.at("kcutoff");                     // reciprocal space cut-off
```

Comment:
This breaks backwards compatibility, so please use something like this:
~~~ cpp
if (j.count("kcutoff") {
    faunus_logger->warn("`kcutoff` is deprecated, use `ncutoff` instead");
    n_cutoff = j.at("kcutoff");
} else {
    n_cutoff = j.at("ncutoff");
}
~~~
Also, please keep the `kcutoff` parameter in the schema for now. Why is `ncutoff` btw. better than `kcutoff`?

Reply:
n is a vector of integers, k=2*pi*n/L a vector of real numbers. Those two often get mixed up.

Reply:
ahh, ok. Should't then `n_cutoff` be an `int`? (it's currently a `double` and also in the schema it should probably be `integer` instead of `number`)

Reply:
Each element in the n-vector is an integer. However, as the length of a general n-vector is a double, for example n=(2,1,0) gives the double sqrt(5), the cutoff for the n-vector should also be allowed to be a double. Or no?

Reply:
I think it's ok with a double then; in the code it's rounded with `ceil` anyways